packages/odyssey-react-mui/src/labs/FileUpload.tsx

```diff
@@ -194,7 +203,7 @@ const FileUpload = ({
       id,
       labelElementId,
     }: RenderFieldComponentProps) => {
-      const fileNames = filesToUpload.map((file) => file.name);
+      // const fileNames = filesToUpload.map((file) => file.name);
```

Comment:
Did we mean to comment this out?

Reply:
Hmm, possible merge conflict thing. Will remove